File: 2003/IntroToVBNETPart2/Tutorial/Environment.vb
Imports System
Imports System.Collections
Imports Microsoft.VisualBasic

Public Class DisplayEnvironment

	Shared Sub Main ()
		Dim env As IDictionary = Environment.GetEnvironmentVariables() 
		Dim e As IDictionaryEnumerator = env.GetEnumerator () 
		While e.MoveNext ()
			Dim entry As DictionaryEntry = DirectCast ( e.Current, DictionaryEntry )
			Console.WriteLine ( "{0}{1}{2}", entry.Key, vbTab, entry.Value )
		End While
	End Sub
End Class

File: 2003/IntroToVBNETPart2/Tutorial/Environment.cs
using System ;
using System.Collections ;

public class DisplayEnvironment
{
	public static void Main ()
	{
		IDictionary env = Environment.GetEnvironmentVariables() ;
		for ( IDictionaryEnumerator e = env.GetEnumerator () ; e.MoveNext () ; )
		{
			DictionaryEntry entry = (DictionaryEntry) e.Current ;
			Console.WriteLine ( "{0}\t{1}", entry.Key, entry.Value ) ;
		}
	}
}